src/components/checkboxes/template.njk

```diff
@@ -9,7 +9,10 @@
 
 {#- a record of other elements that we need to associate with the input using
    aria-describedby – for example hints or error messages -#}
-{% set describedBy = "" %}
+{% set describedBy = params.describedBy if params.describedBy else "" %}
+{% if params.fieldset.describedBy %}
```

Comment:
Am I right in thinking there's no difference between passing `{ fieldset : { describedBy: 'foo' }}` and `{ describedBy: 'foo' }`? Wondering if we need to support both of those, or if we should just support passing it on the fieldset.

Reply:
This would also be consistent with the date input and radio components.

Reply:
@36degrees It's because checkboxes are special, unlike radios.

>Checkboxes with single option set 'aria-describedby' on input

http://govuk-frontend-review.herokuapp.com/components/checkboxes/with-single-option-set-'aria-describedby'-on-input/preview

So the `describedBy` attribute can be without a fieldset.

Reply:
Gotcha, in which case I think this makes sense, thanks for talking me through it.

I can't see a test that covers this functionality (setting aria-describedby on a single checkbox without a fieldset) – could we add one?

Reply:
You're right! Thanks.

I've extended the existing single checkbox tests and made two new ones.

You can see the new bit pushed here: https://github.com/alphagov/govuk-frontend/compare/8e23ce86235c93a77d54b7ccccb0edad923ef7e5..09a9700814d433925b52a9f6d25848093cacdd61